rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -2222,7 +2221,7 @@ namespace rsx
 		}
 
 		vm::ptr<CellGcmReportData> result = sink;
-		result->value = value;
+		result->value = UINT32_MAX;
```

Comment:
Now we need another option to actually disable ZCULL. It is important for debugging overdraw problems you know. Implement this as a combobox with three options for ZCULL behaviour -> "Default (Hardware)", "Always Pass (-1u)", "Always Fail (0)"